st2actions/st2actions/container/base.py

```diff
@@ -129,10 +129,17 @@ def _do_run(self, runner, runnertype_db, action_db, liveaction_db):
             LOG.debug('Action "%s" completed.' % (action_db.name), extra=extra)
 
             # Always clean-up the auth_token
-            updated_liveaction_db = self._update_live_action_db(liveaction_db.id, status,
-                                                                result, context)
-            executions.update_execution(updated_liveaction_db)
+            try:
+                LOG.debug('Setting status: %s for liveaction: %s', status, liveaction_db.id)
+                updated_liveaction_db = self._update_live_action_db(liveaction_db.id, status,
+                                                                    result, context)
+            except:
+                error = 'Cannot update LiveAction object for id: %s, status: %s, result: %s.' % (
+                    liveaction_db.id, status, result)
+                LOG.exception(error)
+                raise
```

Comment:
Yes. When I was debugging, the action actually succeeded but updating the liveaction failed because of non UTF-8 characters. This log line would have been so useful. Also the LOG.debug line above. So this is intentional. No behavior change intended.